pagopa/io-app
# chore: [IOBP-1028] Mixpanel tracking onboarding card methods events on failure

## Short description
This pull request primarily focus on adding new analytics tracking functions to mixpanel when the onboarding payment methods is failing.

## List of changes proposed in this pull request
- Added new functions to track various onboarding payment method events, including authorization denied, user cancellation, duplicate error, and 3DS error
- Updated the component to use a switch statement for handling different onboarding outcomes and tracking the corresponding analytics events

## How to test
- Try to onboard a payment method
- Try each of these outcomes: `AUTH_ERROR`, `CANCELLED_BY_USER` and `ALREADY_ONBOARDED`
- Check that data are sent correctly to mixpanel
